app/templates/gulpfile.js

```diff
@@ -242,9 +278,9 @@ gulp.task('build', ['optimize', 'images', 'fonts', 'tinymce'], function() {
     subtitle: 'Deployed to the build folder',
     message: 'Running `gulp serve-dist`'
   };
-  del(config.temp);
```

Comment:
Got rid of this to kind of fix #272. You'll see styling get removed, but recovers automatically once gulp build completes..